src/Form/components/grid/Cell.tsx

```diff
@@ -115,6 +116,12 @@ const Cell = ({ node: el, form, flags }: any) => {
           }
           return false;
         });
+      const buttonHasAValue = activeStep.buttons.some(({ properties }: any) => {
+        if (properties.link !== LINK_STORE_FIELD) return false;
```

Comment:
if there are no buttons on a step with the LINK_STORE_FIELD link, wouldn't it always be disabled in this case?